components/raftstore/src/store/fsm/apply.rs

```diff
@@ -913,6 +954,8 @@ where
     trace: ApplyMemoryTrace,
 
     buckets: Option<BucketStat>,
+
+    last_write_seqno: Vec<SequenceNumber>,
```

Comment:
I know we already have fields like `last_flush_applied_index`, but `last_` is really hard to understand. How about `unfinished_write_seqno`?

Reply:
Good idea